src/Overlay.tsx

```diff
@@ -17,9 +17,13 @@ export interface OverlayInjectedProps {
   style: React.CSSProperties;
   'aria-labelledby'?: string;
 
-  arrowProps: { ref: any; style: object };
+  arrowProps: {
+    ref: React.RefCallback<HTMLElement>;
```

Comment:
are we explicitly expecting this to be a ref callback? why can't it just be typed as any valid [ref](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/201779301ed80fd8914ccfb4183401fe05eb44d5/types/react/index.d.ts#L92)?

Reply:
This is typed to match that of upstream react-overlays

https://github.com/react-bootstrap/react-overlays/blob/212ffc2aa72756c4c9a2054bb3093050fe0db75b/src/Overlay.tsx#L41-L44

Reply:
okay well I don't understand why the types for react-overlays is that way either 😛 it should be allowed to pass in any valid refs for that feature.

But this is something we'd have to fix in react-overlays first so for now it makes sense for them to be the same type.

Reply:
Agreed.  I was wondering about the ref typing as well, but didn't have enough time for a deep dive into RO today.

Reply:
@bpas247 

Double checked RO and the type is correct. It always passes a callback ref.

https://github.com/react-bootstrap/react-overlays/blob/212ffc2aa72756c4c9a2054bb3093050fe0db75b/src/Overlay.tsx#L62

https://github.com/react-bootstrap/react-overlays/blob/212ffc2aa72756c4c9a2054bb3093050fe0db75b/src/Overlay.tsx#L137-L140